### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by Janpopan**
2nd try - now only improve documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ SonarQube License-Check
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/6ac787bb79b43e39c367/maintainability)](https://codeclimate.com/github/porscheinformatik/sonarqube-licensecheck/maintainability)
 
-This [SonarQube](http://www.sonarqube.org/) plugin ensures that projects use dependencies with compliant licenses. All dependencies and licenses can be viewed per projects and exported to CSV. This enables a simple governance of dependencies and licenses for the whole organization.
+This [SonarQube](http://www.sonarqube.org/) plugin ensures that projects use dependencies with compliant licenses. All dependencies and licenses can be viewed per projects and exported to Excel XLS. This enables a simple governance of dependencies and licenses for the whole organization.
```
Comment (derkoe) [inline README.md:8]: Actually the format is "Excel 2003 XML Format" (https://en.wikipedia.org/wiki/Microsoft_Office_XML_formats)

File: README.md [hunk 2]
```diff
@@ -40,19 +40,54 @@ should look something like this:
 
 ## Configuration
 
-After booting the SonarQube Server with the License-Check Plugin two new options can be found in the tab
-<b>Administration</b>.
+## General Configuration
+After booting the SonarQube Server with the License-Check Plugin be found in the tab <b>Administration</b>.
+
+* Within the <b>General Settings</b> and <b>License Check</b> you find the settings for the plugin.
```
Comment (derkoe) [inline README.md:46]: Maybe mention that the version under "License Check" is just a nicer user interface for same settings under "General Settings"

File: README.md (hunk 1 above)
Comment (Janpopan) [reply README.md:8]: ok

File: README.md (hunk 2 above)
Comment (Janpopan) [reply README.md:46]: ok - I can also do some sreenshots for this case

**Review by derkoe — APPROVED** [review]

**Pull request merged**